src/layouts/partial-cards.html

```diff
@@ -295,19 +295,12 @@
       return routeMatch ? routeData.view : '';
     }
 
-    computeTitle(views, locationName) {
-      return views && views.length > 0 ? 'Home Assistant' : locationName;
+    computeTitle(locationName) {
+      return locationName || 'Home Assistant';
     }
 
     computeStateName(stateObj) {
-      if (stateObj.entity_id === DEFAULT_VIEW_ENTITY_ID) {
-        if (stateObj.attributes.friendly_name &&
-            stateObj.attributes.friendly_name !== 'default_view') {
-          return stateObj.attributes.friendly_name;
-        }
-        return this.computeLocationName(this.hass);
-      }
-      return window.hassUtil.computeStateName(stateObj);
+      return stateObj.attributes.friendly_name;
```

Comment:
Don't remove using `window.hassUtil.computeStateName(stateObj)` in favor of friendly_name, there is a reason that function exists.

Reply:
it's OK for default_view but not for others? I don't understant why

Reply:
Is it worth going back to my original post to review how I envisaged the change to look?

https://community.home-assistant.io/t/home-assistant-location-name-not-respected/32027/2?u=apt

Reply:
@aptonline I don't think we want even more lines of text in the header